lib/rage/cli.rb

```diff
@@ -3,6 +3,7 @@
 require "thor"
 require "rack"
 require "rage/version"
+require "rage/sync_fiber"
```

Comment:
Requiring this file here means it would also be applied when running the server. Instead, we would only want it to be applied when running the console, i.e. in the `console` method.

Reply:
Fair, updated